src/d3a/d3a_core/redis_connections/aggregator_connection.py

```diff
@@ -71,6 +71,9 @@ def add_batch_finished_event(self, device_uuid, event):
 
     def aggregator_callback(self, payload):
         message = json.loads(payload["data"])
+        if d3a.constants.EXTERNAL_CONNECTION_WEB is True and \
```

Comment:
Why not doing the same for local run?

Reply:
Because local runs do not care about multiple simulations, there is only one simulation running at any time. We could of course change the API client to also populate the config_uuid for the case of Redis (essentially when we are selecting an aggregator via the CLI), if you would like to do that ping me to change it.

Reply:
No no keep it as it is for now, no need to complicate thing for now until multiple collaboration local run is needed.